students/pedro-acuña/personal.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1><b>Pedro Ciudad</b></h1>
+    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f9/Cross_Calatrava.svg/300px-Cross_Calatrava.svg.png" alt ="Cruz de Calatrava"/>
+    <p>Hola, me llamo Pedro y soy de Calzada de Calatrava, un pequeño peublo situado en Ciudad Real.</p>
+    <a href ="http://127.0.0.1:5500/prueba1.html">Visitamos a Grogu</a>
```

Comment:
Remove host. Internal links must be relative absolute URL. change it to '/prueba1.html`